install_config/master_node_configuration.adoc

```diff
@@ -297,3 +297,16 @@ manifest files. This is used by the Kubelet to create pods on the node.
 <3> This is the interval (in seconds) for checking the manifest file for new 
 data. The interval must be a positive value.
 ====
+
+=== Parallel image pulls with Docker 1.9+
+
+If you are using Docker 1.9+, you may consider to enable parallel pulls, as the default is still serial pulls. Please note that there is an issue with data corruption with parallel pulls using Docker before 1.9, but starting with 1.9, the corruption issue has been resolved, so it is safe to switch to parallel pulls.
+
+====
+[source,yaml]
+----
+kubeletArguments:
+  serialize-image-pulls: true <1>
+----
+<1> Change to false to enable parallel pulls.
+====
```

Comment:
Is this the right format for kubeletArguments? I would have thought it would have to be the following:
```
kubeletArguments
  serialize-image-pulls:
  - true
```